libraries/GROOT/GRootObjectManager.cxx

```diff
@@ -267,12 +267,12 @@ TH1 *GRootObjectManager::GetLast1D(TObject *object) {
 
 
 
-TH2    *GRootObjectManager::GetNext2D(TObject *object) { }
-TH2    *GRootObjectManager::GetLast2D(TObject *object) { }
+TH2    *GRootObjectManager::GetNext2D(TObject *object) { return 0; }
+TH2    *GRootObjectManager::GetLast2D(TObject *object) { return 0; }
```

Comment:
I understand these are just place holders, but may as well make them return something for now..Otherwise I get ugly warnings....Mac compiling....